src/Inkluzitron/Migrations/20211229172306_UpdateUnsetBirthdayYear.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Inkluzitron.Data.Entities;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Inkluzitron.Migrations
+{
+    public partial class UpdateUnsetBirthdayYear : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            if (!migrationBuilder.IsSqlite())
```

Comment:
Ona existuje situace, že tam bude něco jiného, než SQLite?

Reply:
Časem™ určitě™. Teď netuším, jestli při vytváření nové DB probíhají všechny migrace.. Každopádně to neuškodí a všechny SŘBD určitě nemají strftime.

Reply:
Myslím že probíhají, protože jinde se nevytváří ty všechny triggers atd...